packages/react-components/src/theming/FluentThemeProvider.tsx

```diff
@@ -48,6 +48,7 @@ const defaultTheme: Theme = {
 
 /** Theme context for library's react components */
 const ThemeContext = createContext<Theme>(defaultTheme);
+const ProviderContext = createContext<boolean>(false);
```

Comment:
suggestion: update the name from just `Provider` as that is super generic.
Perhaps `SingleUsageCheckProvider`